trace-sdk/src/main/java/io/bitrise/trace/utils/log/TraceLog.java

```diff
@@ -31,10 +31,6 @@ private TraceLog() {
    */
   public static synchronized Logger makeAndroidLogger() {
     synchronized (lock) {
-      if (logger != null) {
```

Comment:
I refactored the logger a little bit too - previously if you tried to make the logger a different kind of logger after it was already initialised, it wouldn't change e.g. i created an AndroidLogger but now I wanted a SilentLogger - i would expect it to change. 

Does that sound okay?

Reply:
Yes.